playwright-database-testing/env/.env.server

```diff
@@ -0,0 +1,5 @@
+LEDGER_SYNC_USERNAME = default_value
+lEDGER_SYNC_PASSWORD = default_value
```

Comment:
create this draft PR because I need these 2 variable need to be setup in Github Secret @satran004